site/public/js/server.js

```diff
@@ -304,17 +299,20 @@ function newEditCourseMaterialsForm(path, this_file_section, this_hide_from_stud
     $("#material-edit-form", form).attr('data-directory', path);
     form.css("display", "block");
 }
-function captureTabInModal(formName){
+function captureTabInModal(formName, resetFocus=true){
+    releaseTabFromModal(formName);
 
-  var form = $("#".concat(formName));
+    var form = $("#".concat(formName));
 
     /*get all the elements to tab through*/
     var inputs = form.find(':focusable').filter(':visible');
     var firstInput = inputs.first();
     var lastInput = inputs.last();
 
     /*set focus on first element*/
-    firstInput.focus();
+    if(resetFocus){
```

Comment:
When would you not want the focus to land on the first element in the modal?

Reply:
In some modals, the submit button is disabled at the start. This means you cannot tab to it. However when you enable it, you must rerun this function so that tabbing to the submit button works. However in that situation you want to keep the focus on the current item because that is already in the modal. The goal is just to make the submit button tabable again. An example of this is the new student button on the manage students page.